model/patch/patch.go

```diff
@@ -763,15 +763,9 @@ func (p *Patch) IsGithubPRPatch() bool {
 	return p.GithubPatchData.HeadOwner != ""
 }
 
-// IsMergeQueuePatch returns true if the the patch is part of any commit queue:
-// either Evergreen's commit queue or GitHub's merge queue.
+// IsMergeQueuePatch returns true if the the patch is part of GitHub's merge queue.
 func (p *Patch) IsMergeQueuePatch() bool {
-	return p.Alias == evergreen.CommitQueueAlias || p.IsGithubMergePatch()
-}
-
-// IsGithubMergePatch returns true if the patch is from the GitHub merge queue.
-func (p *Patch) IsGithubMergePatch() bool {
-	return p.GithubMergeData.HeadSHA != ""
+	return p.Alias == evergreen.CommitQueueAlias || p.GithubMergeData.HeadSHA != ""
```

Comment:
Do we still have to keep evergreen.CommitQueueAlias? If yes, can we maybe rename it or add a comment explaining why we need to keep it?